docs/Building_for_Linux_(Debian-Ubuntu).md

```diff
@@ -0,0 +1,16 @@
+To build this project do the following steps:
+1. Install cmake and (lib)**sdl2**(-dev) and git
+> > `sudo apt-get install cmake libsdl2-dev git`
+
+2. Download the repo & cd into it
+> > `git clone --recurse-submodules https://github.com/TheOpenSpaceProgram/osp-magnum.git`
+
+> > `cd osp-magnum`
+3. Run Cmake script & compile:
+> > `cmake -DCMAKE_BUILD_TYPE=Debug -B build`
+
+> > `cmake --build build --parallel --config Debug`
+4. Run OSP:
+> > `cd build/bin`
+
+> > `./osp-magnum`
```

Comment:
Why is Github showing already-merged changes? This PR looks way too massive with all these. 

https://github.com/TheOpenSpaceProgram/osp-magnum/blob/master/docs/Building_for_Linux_(Debian-Ubuntu).md
I checked the MD5, and they're exactly the same.